src/Datadog.Trace.AspNet/TracingHttpModule.cs

```diff
@@ -225,5 +221,26 @@ private void TryClearContext(HttpContext context)
                 Log.Error(ex, "Error while clearing the HttpContext");
             }
         }
+
+        private static void AddHeaderTagsFromHttpResponse(System.Web.HttpContext httpContext, Scope scope)
+        {
```

Comment:
Can't `TracingHttpModule` use `HttpContextHelpers.AddHeaderTagsFromHttpResponse()` instead of duplicating the code?

Reply:
I wanted to place the helper code in the instrumentation assembly since that's where it's mostly used. Unfortunately, the AspNet assembly does not reference that, it only references the base Datadog.Trace assembly, so I figured it could be duplicated for now.

Reply:
Agreed, especially since this issue goes away once #1443 is merged